x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/LocalExecutionPlanner.java

```diff
@@ -273,58 +271,6 @@ private PhysicalOperation planFieldExtractNode(LocalExecutionPlannerContext cont
         return physicalOperationProviders.fieldExtractPhysicalOperation(fieldExtractExec, plan(fieldExtractExec.child(), context));
     }
 
-    /**
-     * Map QL's {@link DataType} to the compute engine's {@link ElementType}.
-     */
-    public static ElementType toElementType(DataType dataType) {
```

Comment:
This was a fun one. `LocalExecutionPlanner`'s class calls `LogManager::getLogger` which gives a NPE during benchmarks. Moving to `PlannerUtils` solves this.